src/components/LoanFinding/LendingCategorySection.vue

```diff
@@ -197,6 +193,11 @@ export default {
 			return `loan-card-${index}`;
 		},
 	},
+	mounted() {
+		window.addEventListener('resize', () => {
```

Comment:
The event listener needs to be removed in `beforeDestroy`. Also, the resize should be throttled. A good example of both is in CPS in the `QuickFiltersController`. Notice how the throttled method needs to be stored in a variable so that the remove event listener can find the method to remove.